packages/RichTextEditing-Tests.package/RichTextEditorTextTests.class/instance/testUS69LinkGetsPasted.st

```diff
@@ -0,0 +1,6 @@
+tests
+testUS69LinkGetsPasted
+
+	Clipboard clipboardText: self url.
+	self textEditor paste.
+	self assert: self standardText size + self url size = self editorText size
```

Comment:
So this just generally tests for clipboard content to be pasted. Makes sense to have a test about that but I don't know if the context of URL is necessary for that

Reply:
It is URL specific because links get detected and receive a different treatment during the paste function. Nevertheless I will add a test to check if general clipboard content gets pasted.